### PR TITLE
import-export

### PR DESCRIPTION
**Pull request by Vinhbb96er**
import-export

### CODE REVIEW

**Comment by Vinhbb96er** [comment]
ready

File: app/Http/Controllers/Admin/TourController.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        //
+    }
+
+    public function importTour(TourImportRequest $request)
+    {
+        $categoriesId = $this->categoryRepository->getCategoriesId()->toArray();
+        Excel::load($request->file_import, function($reader) use ($categoriesId) {
+
+            $error = '';
+            $success = 0;
```
Comment (trangkarol) [inline app/Http/Controllers/Admin/TourController.php]: đưa `0` vào file config settings

File: app/Http/Controllers/Admin/TourController.php [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            foreach ($reader->toArray() as $index => $record) {
+                try {
+                    if (!in_array($record['category_id'], $categoriesId) || $record['time_finish']->lte($record['time_start'])) {
+                        throw new Exception();
+                    }
+                    
+                    $this->tourRepository->create($record);
+                    $success++;
```
Comment (trangkarol) [inline app/Http/Controllers/Admin/TourController.php]: convention

File: app/Models/Booking.php [hunk 3]
```diff
@@ -47,4 +48,31 @@ public function user()
     public function getUpdatedTimeAttribute() {
         return Carbon::parse($this->attributes['updated_at'])->format('l\, F j, Y');
     }
+
+    public function getStatusTextAttribute() {
+        $text = "";
+
+        switch ($this->attributes['status']) {
+            case config('setting.booking_cancel'):
+                $text = "Canceled";
```
Comment (trangkarol) [inline app/Models/Booking.php]: I18n

Comment (trangkarol) [reply app/Models/Booking.php]: sao ko `return "Canceled";`

**Comment by trangkarol** [comment]
commented

**Comment by Vinhbb96er** [comment]
ready

**Pull request merged**